src/card.css

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: min-content;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 16px;
+}
+
+.card__content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+
+  margin: 16px 3px 0;
```

Comment:
that 3px margins looks strange, you used it for aligning? in that case better set auto or use flex